TemporaryOysters/bangazon-cli
# CLI customer saved to DB

##Pull Requests



##Descriptions:
We can now take the information entered in the CLI to create a new customer in our DB.
* Please describe what issue this PR solves











##Steps to Test:
git fetch
git checkout th-CLI-to-DB
delete your existing bangazon.db due to updates.
create new bangazon.db by running main.py in CL
populate DB
-comment out line 7 in main.py & comment in line 9
-run main.py in CL
Run tests
-cd in tests directory
-in CL run: python -m unittest discover -s . -p "Test*.py" -v
-15 tests should pass
Check bangazon.db in DB Browser & make sure you're not seeing any funny business.
-run python CLI.py and create a new customer.
-refresh the DB Browser to see if the new customer is in the DB.
Pat yourself on the back for making it through this PR. Thank you for reviewing.


##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?


* Any command line utilities to run?
